MAUI/TreeMap/levels.md

```diff
@@ -0,0 +1,681 @@
+---
+layout: post
+title: Understanding Levels in .NET MAUI TreeMap | Syncfusion
+description: Learn about the levels within Syncfusion .NET MAUI TreeMap control, exploring its fundamental features and functionalities for effective treemap visualization.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+ 
+# Levels in .NET MAUI TreeMap (SfTreeMap)
+Levels in .NET MAUI TreeMap allow for hierarchical data visualization, enabling multi-level categorization and exploration of complex datasets. You can seamlessly navigate through different levels of data hierarchy, facilitating comprehensive understanding and analysis.
+
+## TreeMap Level
+Using `Levels` property of `SfTreeMap` you can integrate multiple hierarchical data levels to the TreeMap. The `GroupPath` property of `TreeMapLevel` specifies the name of the property in the data object that determines how items in the tree map are grouped. The grouping is based on the values of this property in the data source, which splits the level into distinct groups.
+
```

Comment:
In the following example, three levels are added and each level is configured using the GroupPath.